docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,92 @@
+This guide was developed using a `Nokia G-240W-B` router (software version 3FE47969AGCA21). It should work for other models in the `G` series.
+
+!!! note
+    There is no single way to do it right. Chose the one best suited for you.
+This guide is `IPv4` and `IPv6` compatible, but it is not mandatory to use `IPv6`.
+
+### Prerequisites
+
+- The `Raspberry Pi` is accessible from the `LAN` network via `SSH`.
+
+### Grab your IPv4 and IPv6 address from your Raspberry Pi
+
+Notice that the `IPv4` and `IPv6` addresses suffixes matches the number `xxx.168.1.10` and `xxxx::192:168:1:10` respectively, this is intentional to recognize the addresses were manually assigned in the `Raspberry Pi`.
```

Comment:
I don't understand the second part of this sentence. What do you want to say here

`.... this is intentional to recognize the addresses were manually assigned in the `Raspberry Pi`.`

Reply:
Removing that part since I don't want to confuse users.